src/fetcher/fetcher.mts

```diff
@@ -21,30 +21,6 @@ const sendBody = (method: Method) =>
   method === 'patch' ||
   method === 'delete';
 
-function queryString(params: Record<string, unknown>): string {
```

Comment:
All the fetcher code is really weird and overcomplicated. I don't want to touch too much of it but I'm gonna dare to make this one change :sweat_smile: